src/contracts/core/AllocationManager.sol

```diff
@@ -605,6 +656,41 @@ contract AllocationManager is
         return uint256(int256(int128(uint128(a)) + b)).toUint64();
     }
 
+    /// @dev Reverts if the operator set does not exist.
+    function _checkIsOperatorSet(
+        OperatorSet memory operatorSet
+    ) internal view {
+        require(_operatorSets[operatorSet.avs].contains(operatorSet.id), InvalidOperatorSet());
+    }
+
+    /// @dev Reverts if the provided arrays have different lengths.
+    function _checkArrayLengthsMatch(uint256 left, uint256 right) internal pure {
+        require(left == right, InputArrayLengthMismatch());
+    }
+
+    /// @dev Reverts if the operator is not registered.
+    function _checkIsOperator(
+        address operator
+    ) internal view {
+        require(delegation.isOperator(operator), InvalidOperator());
+    }
+
+    function _checkAVSMetadata(
+        address avs
+    ) internal view {
+        require(_avsRegisteredMetadata[avs], NonexistentAVSMetadata());
+    }
```

Comment:
Request - can we rename this to `_checkAVSExists`? Or at least comment that this is an existence check?